### PR TITLE
fix test: IterateRegistryAsTargetTestCase

### PR DESCRIPTION
**Pull request by sajinieKavindya**
Fix IterateRegistryAsTargetTestCase failure as part of the test hackathon

### CODE REVIEW

File: integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/config/sequences/iterate/iterateLogAndSendSequence [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sequence xmlns="http://ws.apache.org/ns/synapse" name="iterateLogAndSendSequence">
+	<in>
```
Comment (Nirothipan) [inline integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/config/sequences/iterate/iterateLogAndSendSequence]: shall we avoid the usage of in and out mediators

File: integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/sequences/iterateLogAndSendSequence.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (arunans23) [inline integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/sequences/iterateLogAndSendSequence.xml]: add licence headers

Comment (sajinieKavindya) [reply integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/sequences/iterateLogAndSendSequence.xml]: done

File: integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/config/sequences/iterate/iterateLogAndSendSequence (hunk 1 above)
Comment (sajinieKavindya) [reply integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/config/sequences/iterate/iterateLogAndSendSequence]: done

**Pull request merged**